LiskHQ/lisk-core
# Missing genesis assets for interoperability module

### What was the problem?

This PR resolves #729 

### How was it solved?

- [x] Added genesis assets for `interoperability` module for both `devnet` and `alphanet`
- [x] Regenerated the genesis blocks
- [x] Fix method invocation for `setValidatorBLSKey` method
- [x] Regenerate the genesis blocks for devnet and alphanet

### How was it tested?

- Started a devnet mainchain (Lisk Core) node (chainID: `04000000`)
  - `interoperability_getChainAccount` endpoint returns valid result
  - Registered the sidechain on the mainchain
  - Register the `blsKey`, `proofOfPossession` and `generatorKey` for all the init validators
  - Wait until the `blsKey` is updated (non-zeros) for all the validators when querying `consensus_getBFTParameters` endpoint
- Bootstrapped a sidechain node with chainID (`04000001`)
  - Register the mainchain on the sidechain
- Check the events for chain registration confirmation
